one_fm/public/js/doctype_js/purchase_order.js

```diff
@@ -84,112 +81,6 @@ var hide_tax_fields = function(frm) {
 var hide_subscription_section = function(frm) {
 	frm.set_df_property('subscription_section', 'hidden', true);
 };
-var confirm_accept_approve_purchase_order= function(frm) {
```

Comment:
I hope we need OTP verification, we dont need to update the signature from employee to the PO

Reply:
Are you sure? because we applied the verification part just because we had to fetch the signature.